### DISCUSSION TITLE
Anyone have a fix for the internet connection-killing problem?

### DISCUSSION THREAD

**Discussion by amg433** [kubilus1/autoortho#296] [General]
People keep asking about this, but no one seems to have a solution. The least I can do is use a VPN, which allows me to load into Australia, but all other regions don't load.

Am I screwed because of my ISP?

**Comment by dpdurst** [comment]
To answer your question, its not an AO issue if your ISP is throttling due to the amount of data being streamed. That is between you and how your ISP handles that kind of data.

**Reply by amg433** [reply]
For what it's worth, I'll try 0.5.0. Maybe I'll get lucky. If it is indeed an ISP issue, then I guess I'll go back to using old-fashioned ortho.

**Reply by amg433** [reply]
No luck. I give up.
